pkg/reconciler/managed/reconciler.go

```diff
@@ -657,10 +657,10 @@ func WithRecorder(er event.Recorder) ReconcilerOption {
 	}
 }
 
-// WithManagementPolicies enables support for management policies.
-func WithManagementPolicies() ReconcilerOption {
+// WithoutManagementPolicies disables support for management policies beta feature.
+func WithoutManagementPolicies() ReconcilerOption {
```

Comment:
See the above comment. I don't think we would need any change other than making the value of [this flag](https://github.com/upbound/provider-aws/blob/6bbeee5ef06a4c83eda9f601678b2f425ba082c7/hack/main.go.tmpl#L50) `true` by default.